app/assets/javascripts/discourse/routes/user-activity-index.js.es6

```diff
@@ -1,14 +1,5 @@
 import UserActivityStreamRoute from "discourse/routes/user-activity-stream";
 
 export default UserActivityStreamRoute.extend({
-  userActionType: undefined,
-
-  actions: {
-    didTransition() {
-      this._super();
-      this.controllerFor("user").set("indexStream", true);
-      return true;
-    }
-  }
-
+  userActionType: undefined
```

Comment:
I prefer `null` when dealing with uninitialized values. `undefined` is good for something that you didn't assign a value, like when you do `let x`.